lib/src/components/NxList/types.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { HTMLAttributes, LiHTMLAttributes, WeakValidationMap, ReactElement,
+  ReactNode, ButtonHTMLAttributes, AnchorHTMLAttributes } from 'react';
+import PropTypes from 'prop-types';
+
+export type NxListProps = HTMLAttributes<HTMLUListElement> & {
+  bulleted?: boolean;
+  children?: ReactElement | ReactElement[] | null | ReactNode;
```

Comment:
Is this not the same as just `ReactNode`?  `ReactNode` is already a union of a bunch of different stuff

Reply:
> Is this not the same as just `ReactNode`? `ReactNode` is already a union of a bunch of different stuff

Yes, this was an error on my end. It has been changed to use `ReactNode`
https://github.com/sonatype/sonatype-react-shared-components/pull/451/files#diff-acef640fb1686ef46c36378eb99744a098e85ffac1cacbcea722e14f8e0f0ce3R10-R15

Reply:
> Yes, this was an error on my end. It has been changed to use ReactNode
https://github.com/sonatype/sonatype-react-shared-components/pull/451/files#diff-acef640fb1686ef46c36378eb99744a098e85ffac1cacbcea722e14f8e0f0ce3R10-R15

Closer, but `ReactNode` includes `null` too so that part is still redundant.

Reply:
> > Yes, this was an error on my end. It has been changed to use ReactNode
> > https://github.com/sonatype/sonatype-react-shared-components/pull/451/files#diff-acef640fb1686ef46c36378eb99744a098e85ffac1cacbcea722e14f8e0f0ce3R10-R15
> 
> Closer, but `ReactNode` includes `null` too so that part is still redundant.

Fixed.
ce12545a1c0ea836a797e3c2e8aa4cc2d177c09e